Hello Microverse Project

demesameneshoa opened the pull request.

demesameneshoa committed:
Update README.md

usorfaitheloho commented on README.md:
```diff
@@ -1,3 +1,238 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
```
please kindly remove all commented lines of code as it takes up unnecessary space, causes confusion and might lead to maintenance issues 😄 . 
`

usorfaitheloho commented on README.md:
```diff
@@ -1,3 +1,238 @@
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
You did well using the required Microverse Readme template for this project, Please remove all commented lines of code, and all unnecessary descriptive messages to make your code legible 😄 and also write the details of each section of your README file as it relates to the project

usorfaitheloho reviewed (changes requested):
Hi @demesameneshoa    , 

>You are almost there!
Great effort so far, but unfortunately there are some issues that you need to review in order to get your project approved, but you got this!


![Alt Text](https://camo.githubusercontent.com/59fa9d95c18dbd5a8e47490a9e4b48393f569685410f3c78d4d83a9764fa1392/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f74385950663341434b656f754c7a454976722f67697068792e676966)
>


## Highlights
- Well-organized files :100: 
- Linters are passing :100: 
<br/>
<br/>
<br/>

## Required Changes ♻️
- [x] Please add a descriptive Pull Request Title and summary to give a better understanding of the changes you introduced in this milestone.

- [x] kindly ensure that you use the right  [Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) to enable good management of your repository

- [x]  When using the [Microverse Readme template](https://github.com/microverseinc/readme-template) remember to fill it with the appropriate content relative to the work you did for the project

- [x] As part of the requirements, you should add `test.md` and `.gitignore` files to your project. 

- [x] As part of the requirement, you should add an `HTML` and `CSS` file with these contents to enable you to view the `html` contents in the browser.

![test md](https://user-images.githubusercontent.com/20537712/234341648-7375dba6-babb-4462-bd94-337881fac727.png)



 


_Check the comments below to see technical issues found during this review._

## Optional suggestions

- [x] Please try to use a descriptive branch name and commit messages in your project 😄 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
<br/>


Cheers and Happy coding!👏👏👏
<br/>
<br/>
<br/>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

demesameneshoa committed:
Update README.md

Updated the readme file as per the code review comment

demesameneshoa commented:
- Updated the readme file
- Added the index.html, style.css and.git ignore files

iambenkis commented on README.md:
```diff
@@ -1,3 +1,36 @@
+
+# Hello Microverse Project
+
+> Training on github practice
+
+
+## Built With
+
+- Html & CSS
+
+## Authors
+
+👤 *Demes Ameneshoa*
+
+- GitHub: [@demesameneshoa](https://github.com/demesameneshoa)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Micronautes
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
- [ ] As requested by the previous CR make user you use the[ MICROVERSE README TEMPLATE](https://github.com/microverseinc/readme-template) , then you should customize it with information about your project.
- [ ] Kindly avoid broken link in your README file 😄

iambenkis reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @demesameneshoa   
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

- [ ] [**OPTIONAL**] I Noticed that you wanted to merge branches from `feature` branch to the `main` branch, note that this is not the proper way of doing that , it should be from `feature` to `main` instead. Next to make sure you take care of that point 😄 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

demesameneshoa committed:
Update README.md

TadesseDev commented on README.md:
```diff
@@ -1,3 +1,224 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
```
- [x] Dear @demesameneshoa, I have seen the previous reviewers have made a suggestion to update your readme file content with related information. As part of that, I have left you some comments to help you address them. Remember, the template is designed by Microverse for us all to use to professionally document our projects and have it filled with content related to each project we may be working on.

TadesseDev commented on README.md:
```diff
@@ -1,3 +1,224 @@
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
```
- [ ] Please consider Updating the Microverse logo with yours if possible or remove it totally. The template is designed by Microverse for us all to use to professionally document our projects and has it filled with content related to each project we may be working on.

demesameneshoa replied:
@TadesseDev There is no logo in the readme file, is it mandatory to have a logo?

TadesseDev replied:
Sorry, @demesameneshoa, I must have to mismatched it with another project. Please address the other changes and request a review. You got this :hand:

TadesseDev commented on README.md:
```diff
@@ -1,3 +1,224 @@
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
```
- [ ] Please notice the section in your readme which start with `>` are directions/guidelines from microverse to help you understand what to write in that section and should not be part of your readme file. Please consider removing this and all the like contents from the readme file.

TadesseDev commented on README.md:
```diff
@@ -1,3 +1,224 @@
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
```
- [ ] Please consider removing the `Live demo` section from the readme file. It is only required when you have a project that is deployed.

TadesseDev reviewed (changes requested):
### Hi @demesameneshoa 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7](https://user-images.githubusercontent.com/69077061/212385281-4797c58a-d358-423a-aa7c-832c22203865.gif)




## Highlights
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
 - [OPTIONAL] I Noticed that you wanted to merge branches from the feature branch to the main branch, note that this is not the proper way of doing that, it should be from the feature to the main instead. Next, make sure you take care of that point :+1: 
- [Optional] Please try to be more descriptive in your pull request sumery. This is a place where you can introduce what changes you have made since the last merge. You can also specify any information that you would like to communicate with the code reviewer of the project.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

demesameneshoa committed:
Update README.md

Removed comments

demesameneshoa committed:
Update README.md

Updated readme file as per the code review
- removed comments
- updated the content and added content related to the project

demesameneshoa committed:
Update README.md

Updated Readme file

demesameneshoa committed:
Update README.md

Removed license broken link

demesameneshoa committed:
Update README.md

demesameneshoa committed:
Update README.md

demesameneshoa committed:
Update README.md

divinecharlotte reviewed (approved):
# Hello @demesameneshoa   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive commit messages 👍 


  ☠️ Next time make sure that you add a Pr description and put into consideration that it's always mandatory I can see that the previous reviewer mentioned it as an OPTIONAL suggestion which might be the reason you didn't focus on it that much  (that's because the first reviewer missed it out sorry) and adding a PR description will make your Pr more descriptive and easy to understand what you did in that Pr for the people reviewing your code 


## Congratulations! 🎉 

<img src="https://media3.giphy.com/media/3KC2jD2QcBOSc/giphy.gif"/>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
